Platforms/QemuQ35Pkg/QemuQ35Pkg.dsc

```diff
@@ -924,6 +925,8 @@
   !include CryptoPkg/Driver/Bin/CryptoDriver.inc.dsc
 !endif
 
+QemuQ35Pkg/Library/PeiFvMeasurementExclusionLib/PeiFvMeasurementExclusionLib.inf

```

Comment:
Should this be gated by TPM_REPLAY_ENABLED? Saves some build time (not much), but also may trigger reporting tools that check for built binaries being used?

Reply:
I decided to keep this one in the normal platform build. It has a bit more platform touch points (like FV PCDs) than code being built in TpmTesting so I didn't want it to get broken and not discovered until the next person turns on the feature.